catlearn/ga/algorithm.py

```diff
@@ -48,13 +48,23 @@ def __init__(self, population_size, fit_func, features, targets,
             performed.
         """
         # Set parameters.
+        self.nprocs = nprocs
         self.step = -1
-        self.population_size = population_size
         self.fit_func = fit_func
         self.dimension = features.shape[1]
         self.nsplit = nsplit
         self.accuracy = accuracy
 
+        if population_size is None:
```

Comment:
If set to `'auto'` we should perhaps try and come up with some better initialization for `population_size`. I think the performance will be better if we can have `n * nprocs` in the majority of cases, particularly if we run them in an intelligent manner. Either way, a population size of 8 is probably unnecessarily small.

Reply:
Done. What do you think of `population_size = 2*nprocs*((8 // nprocs) + 1)` as a default? It never gets below 18.